README.md

```diff
@@ -42,6 +42,49 @@ You can also use it as a library within your existing logic.
 cargo-run-bin = { version = "1.7.2", default-features = false }
 ```
 
+### Using wrapper
+
+Installing with a minimal wrapper and an alias. 
+
+```sh
+cd my/rust/project
+echo ".bin/" >> .gitignore
+cargo new --bin cargo-bin
```

Comment:
Rather than recommending polluting a projects root directly, let's move it in to a more common `tools` folder instead. :)

```suggestion
cargo new --bin tools/cargo-bin
```